### DISCUSSION TITLE
Error on class rebuild

### DISCUSSION THREAD

**Discussion by neha04** [pimcore/pimcore#16067] [Q&A / Support]
Hi All,

I am using pimcore 10.5.20 and while geting class rebuild command i am getting following issue.

[Doctrine\DBAL\Exception\DriverException]
An exception occurred while executing 'CREATE TABLE IF NOT EXISTS object_query_Acquisitions (
oo_id int(11) UNSIGNED NOT NULL default '0',
oo_classId varchar(50) default 'Acquisitions',
oo_className varchar(255) default 'Acquisitions',
PRIMARY KEY (oo_id),
CONSTRAINT fk_object_query_Acquisitions__oo_id FOREIGN KEY (oo_id) REFERENCES objects (o_id) ON DELETE CASCADE
) DEFAULT CHARSET=utf8mb4;':

SQLSTATE[HY000]: General error: 1826 Duplicate foreign key constraint name 'fk_object_query_Acquisitions__oo_id'

**Comment by dpfaffenbauer** [comment]
Check if your database is using lower_case_table_names or not. I just had the same issue

**Reply by neha04** [reply]
Variable_name: lower_case_table_names
        Value: 0

**Reply by dpfaffenbauer** [reply]
Where? The System you exported the database from or where you imported it into?

**Reply by neha04** [reply]
the system i imported into

**Reply by dpfaffenbauer** [reply]
Then please also check the system you got the database from. It should have the same setting, otherwise the database created all tables lower-case and you're imported it into a database where its using camel-case.
